KubeArmor/config/config.go

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package config
+
+import (
+	"os"
+
+	"flag"
+
+	kg "github.com/kubearmor/KubeArmor/KubeArmor/log"
+	"github.com/spf13/viper"
+)
+
+type KubearmorConfig struct {
+	Grpc           string // GRPC Port to use
+	Cluster        string // Cluster name to use for feeds
+	LogPath        string // Log file to use
+	HostVisibility string // Host visibility to use for kubearmor in process mode
+	Policy         bool   // Enable/Disable policy enforcement
+	HostPolicy     bool   // Enable/Disable host policy enforcement
+	KVMAgent       bool   // Enable/Disable KVM Agent
+}
+
+var GlobalCfg KubearmorConfig
+
+const CFG_CLUSTER string = "cluster"
+const CFG_GRPC string = "gRPC"
+const CFG_LOGPATH string = "logPath"
+const CFG_HOSTVISIBILITY string = "hostVisibility"
+const CFG_KUBEARMORPOLICY string = "enableKubeArmorPolicy"
+const CFG_KUBEARMORHOSTPOLICY string = "enableKubeArmorHostPolicy"
+const CFG_KUBEARMORVM string = "enableKubeArmorVm"
```

Comment:
```suggestion
const CFG_KUBEARMORPOLICY string = "policy"
const CFG_KUBEARMORHOSTPOLICY string = "host-policy"
const CFG_KUBEARMORVM string = "vm"
```
Since the flags are for kubearmor only, I don't think they need to have it. Also POSIX suggests to have only lowercase flag names.
We can use `KUBEARMOR` prefix for environment variables like you have for the config file using
`v.SetEnvPrefix("KUBEARMOR")`

Reply:
I didn't want to change existing command line parameter key labels (at least in this PR). If we change those then there changes to be done in tests/documentation etc. I started off making changes that way and then reverted back because of these deps.

Reply:
@nyrahul can you handle the go-lint warnings (e.g., adding single comments or go-lint ignoring tag on top of each line)?